v20.1/import-into.md

```diff
@@ -100,7 +100,7 @@ To import a local file, you have the following options:
 - Option 2. Make the file accessible from each local node's store:
     1. Create an `extern` directory on each node's store. The pathname will differ depending on the [`--store` flag passed to `cockroach start` (if any)](cockroach-start.html#general), but will look something like `/path/to/cockroach-data/extern/`.
```

Comment:
the "on each node" part goes away as does step 2, but in fact, i think we could just say to to use `cockroach nodelocal upload data.sql` (or manually place the file in the `extern` directory).